doc/user_manual/chIntroduction.tex

```diff
@@ -103,9 +103,19 @@ \section{Evolution of SixTrack}
     \item \textbf{Version 4}
         \todo[inline]{Add stuff}
     \item \textbf{Version 5}
-        \todo[inline]{Add stuff}
+        Includes a full rewrite of the input parsing, upgrade of the source to Fortran 2008, and makes all major arrays dynamically allocatable.
+        Machine size and particle numbers are no longer defined by compiler flags.
```

Comment:
... but the correct amount of memory is allocated on the fly. Arbitrary number of particles are supported.